Engine/Scripting/Enemy.cpp

```diff
@@ -204,6 +211,23 @@ void Enemy::Init()
 	}
 }
 
+void Enemy::GetParalyzed(float percentage)
+{
+    mIsParalyzed = true;
+    mSpeed *= percentage;
+    mCurrentParalyzedTimer = mParalyzedTimer;
+    mParalysisSeverityLevel = percentage;
+}
+
+void Enemy::GetParalysisCured(float percentage)
+{    
+    mIsParalyzed = false;
+    mSpeed /= percentage;
+    mCurrentParalyzedTimer = 0;
+
+    mParalysisSeverityLevel = 1.0f;
+}
+
```

Comment:
Instead of two methods use one and pass the percentage and a boolean and do not use Get word if this is not a Getter. Call it Paralyze() instead